contracts/VToken.sol

```diff
@@ -1349,6 +1658,55 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         emit NewMarketInterestRateModel(oldInterestRateModel, newInterestRateModel);
     }
 
+    /**
+     * @notice accrues interest and updates the stable interest rate model using _setStableInterestRateModelFresh
+     * @dev Admin function to accrue interest and update the stable interest rate model
+     * @param newStableInterestRateModel the new interest rate model to use
+     * @return uint 0=success, otherwise a failure (see ErrorReporter.sol for details)
+     */
+    function setStableInterestRateModel(StableRateModel newStableInterestRateModel) public override returns (uint256) {
+        bool canCallFunction = AccessControlManager(accessControlManager).isAllowedToCall(
+            msg.sender,
+            "setStableInterestRateModel(address)"
+        );
+
+        // Check if caller has call permissions
+        if (!canCallFunction) {
+            revert SetStableInterestRateModelOwnerCheck();
+        }
+        accrueInterest();
+        // _setInterestRateModelFresh emits interest-rate-model-update-specific logs on errors, so we don't need to.
+        return _setStableInterestRateModelFresh(newStableInterestRateModel);
+    }
+
+    /**
+     * @notice updates the stable interest rate model (*requires fresh interest accrual)
+     * @dev Admin function to update the stable interest rate model
+     * @param newStableInterestRateModel the new stable interest rate model to use
```

Comment:
```suggestion
     * @param newStableInterestRateModel The new stable interest rate model to use
```

Reply:
Fixed, can check the changes at [this commit](2981e0d08b247b7c7a7dc3461e9701d7d593c520).